packages/aws-cdk/lib/cli/cli.ts

```diff
@@ -454,6 +454,15 @@ export async function exec(args: string[], synthesizer?: Synthesizer): Promise<n
         ioHost.currentAction = 'notices';
         return cli.acknowledge(args.ID);
 
+      case 'cli-telemetry':
+        ioHost.currentAction = 'cli-telemetry';
```

Comment:
Seems unnecessary because of:

https://github.com/aws/aws-cdk-cli/blob/7cac5d2d828800b500f80fe00aa0c0ab885b56c5/packages/aws-cdk/lib/cli/cli.ts#L183

But since all other commands are doing it, ok.